src/sidebar/styles.css

```diff
@@ -109,6 +110,7 @@ footer {
   display: flex;
   flex: 0 0 40px;
 }
+*/
```

Comment:
Are we not using this selector anymore? Should we just ✂️ it?

Reply:
👍